src/browser/modules/Stream/CypherFrame/AsciiView.tsx

```diff
@@ -122,6 +126,16 @@ export class AsciiViewComponent extends Component<
     this.props.setAsciiMaxColWidth(maxColWidth)
   }
 
+  /**
+   * Replaces newline characters, with a double \\ to escape newline in render
+   */
+  removeNewlines(serializedRows: SerializedRows): SerializedRows {
+    const newSerializedRows: SerializedRows = serializedRows.map(row => {
+      return row.map(value => value.replace('\n', '\\n'))
```

Comment:
In the rest of the codebase, we've not aliased simple types, so I'd personally prefer if we left it as string[][]. You also only need to write the type on the input argument, so it's not needed to specify it on row 133 as it's inferred.